homeassistant/components/traccar/device_tracker.py

```diff
@@ -172,11 +159,24 @@ def __init__(self, api, hass, async_see, scan_interval,
             for custom_attr in self._custom_attributes:
                 if device_info.get(custom_attr) is not None:
                     attr[custom_attr] = device_info[custom_attr]
+                    if custom_attr in self._skip_accuracy_on:
+                        skip_accuracy_filter = True
```

Comment:
Since this is part of the loop it will set the value for the last item in that loop.

Reply:
The goal is to set ```skip_accuracy_filter``` var to true when at least one of the items of ```_custom_attributes``` are contained on ```_skip_accuracy_on``` collection and are present in traccar message.
ie. Position has an higher accuracy but message contains alarm attribute, so it's importatn to receive despite of accuracy.